### PR TITLE
Feature/new design responsive

### PR DESCRIPTION
**Pull request by cbrzn**
# Short description
App is not responsive right now

# How can it be tested
Go to the app and check it w/mobile view

# Tracker ID
https://linear.app/aragon/issue/VOT-204/main-layout
https://linear.app/aragon/issue/VOT-172/v0-for-mobile-version

### CODE REVIEW

**Comment by WissenIstNacht** [comment]
There's a few things that don't work properly, like the search bar on the register token page. Let's collect them on linear and move forward from there.

**Review by WissenIstNacht — APPROVED** [review]

**Comment by c0rv0s** [comment]
looks super good...whats up with all the token cards saying 7 active proposals and 987m mcap?

File: pages/tokens/info.tsx [hunk 1]
```diff
@@ -68,25 +86,33 @@ const EmptySection = styled.div`
   background: #eef4fb;
   border-radius: 13px;
   text-align: center;
+  font-size: 18px;
+
+  @media ${({ theme }) => theme.screens.tablet} {
+    padding: 0;
```
Comment (WissenIstNacht) [inline pages/tokens/info.tsx:92]: Some padding might be good here. Not sure how much though 😅

**Review by WissenIstNacht — COMMENTED** [review]
Also the button to remove a question overlaps with text on the process creation page. Might be my bad though. Added [this ]( https://github.com/vocdoni/bridge-ui/blob/b8ec69bb1443b35494ab15d76a828446b4c5a7e5/pages/processes/new.tsx#L469) (probably to try something out).

**Review by WissenIstNacht — COMMENTED** [review]
Hey sorry cesar for being pedantic but there's two more things 😁 
1) the token list is not aligend with the title on the "find token page" when viewing in desktop mode (it's slightly centered w.r.t. the parent component). I personally think it looks better as it's done in this PR, but don't know how adri feels about it.
2) You have the hard-coded "7 active proposals" text on the cards

**Review by nestorbe — APPROVED** [review]

**Pull request merged**